client/src/components/navbar/Navbar.tsx

```diff
@@ -8,7 +8,7 @@ const Navbar = observer(() => {
   const handleLogout = () => {
     authStore.logout();
   };
-  const navbarClass = authStore.isLogged ? 'navbar logged' : 'navbar';
+  const navbarClass = authStore.isLogged ? "navbar logged" : "navbar";
   return (
     <div className={navbarClass}>
       {authStore.isLogged && (
```

Comment:
You dont have to repeat 'navbar' class twice
```suggestion
  const navbarClass = authStore.isLogged ? "logged" : "";
  return (
    <div className={`navbar ${navbarClass}`}>
      {authStore.isLogged && (
```